fv/iptables_lock_test.go

```diff
@@ -0,0 +1,112 @@
+// +build fvtests
+
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fv_test
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("with running container", func() {
+	var containerIdx int
+	var containerName string
+	var felixCmd *exec.Cmd
+
+	cmdInContainer := func(cmd ...string) *exec.Cmd {
+		arg := []string{"exec", containerName}
+		arg = append(arg, cmd...)
+		return exec.Command("docker", arg...)
+	}
+
+	BeforeEach(func() {
+		containerName = fmt.Sprintf("felix-fv-%d-%d", os.Getpid(), containerIdx)
+		containerIdx++
+		felixDir, err := os.Getwd()
```

Comment:
Do these tests need felixDir to be correct?  I would expect Getwd() here to give `<reporoot>/fv`, not `<reporoot>` - and I'm not sure that is what you intend with the name `felixDir`.